lib/functions_framework/legacy_event_converter.rb

```diff
@@ -129,12 +130,20 @@ def construct_cloud_event context, data
                                time:              context[:timestamp]
     end
 
-    def convert_source service, resource
+    def convert_source service, resource, domain
       return ["//#{service}/#{resource}", nil] unless CE_SERVICE_TO_RESOURCE_RE.key? service
 
       match = CE_SERVICE_TO_RESOURCE_RE[service].match resource
       return [nil, nil] unless match
-      ["//#{service}/#{match[1]}", match[2]]
+
+      if service == "firebasedatabase.googleapis.com"
+        raise "Invalid firebasedatabase event: domain is nil" if domain.nil?
```

Comment:
Please just `return [nil, nil]` instead, to signal that we do not recognize the event format and that it should not be converted. (If we were to raise an exception here that is due to _our_ issue of sending a malformed event, it's not actionable by a user of the framework.)

Reply:
done